shop/shop/src/main/java/ro/msg/learning/shop/dtos/OrderDTO.java

```diff
@@ -0,0 +1,23 @@
+package ro.msg.learning.shop.dtos;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class OrderDTO {
+    private Integer id;
+    private List<OrderDetailDTO> orderDetailDTOList;
```

Comment:
Instead of `orderDetailDTOList`, name this field `orderDetails` even if its type is `List<OrderDetailDTO>`. The one calling the API does not care if in our implementation the field is a DTO or not. It should only have a suggestive name.